src/components/GiftCard.vue

```diff
@@ -30,7 +30,22 @@ import genQrCode from './GenQRCode'
 import {date} from 'quasar'
 export default {
   props: {
-    giftCard: [Object],
+    code: {
+      type: String,
+      default: 'xxx',
+    },
+    amount: {
+      type: Number,
+      default: 0,
+    },
+    expiry: {
+      type: Number,
+      default: 0,
+    },
+    createdDate: {
+      type: String,
+      default: new Date().toDateString(),
```

Comment:
createdDate should not be a String, can choose between (timestamp (Number) getTime() or a Date object)
Date stored as String with no formatting rules wont be stable